### DISCUSSION TITLE
Fail to transfer domain

### DISCUSSION THREAD

**Discussion by Mic75** [vercel/community#4714] [Help]
### Summary

Hello Vercel Community,
I am unsuccessfully attempting to transfer one of my domain  to another Vercel user account.

### Steps to Reproduce

I've tried two methods: 
- Move to another Vercel Account
- Transfer out 

The first one is sending a mail to the destination user, when he clicks the link it says that the token is expired.

For the second one: when I share the auth code with him and he tries to move in the domain using the auth code, there is an error appearing in the form, saying that the domain is already attached to another account.

I carefully followed Vercel's [guide](https://vercel.com/docs/projects/domains/transfer-your-domain) step-by-step but it's not working in my case.

Could you please provide further guidance

**Comment by amyegan** [comment]
Hi @Mic75. It sounds like you're trying to move the domain to another account, so the steps to [transfer a domain to another Vercel user or Team](https://vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-another-vercel-user-or-team) should be the ones to follow. 

How long after you sent the request did your friend try to click the domain transfer acceptance link? If the email sat in their inbox for a while, the link may legitimately have expired. If that's the case, you can follow the same steps again to send a new link to them.

**Reply by Mic75** [reply]
Hi @amyegan ,

Thanks for your answer.

So my friend tried to click the link right after I sent the request. It arrived in his box within a minute or so.
I've been through the steps a couple of time, he received the link each time, but with the same error when clicking the link.

**Reply by cimply42** [reply]
Hi @amyegan,

@Mic75 is trying to transfer the domain to me. As he said, when i click the link (which i do within a few minutes of him sending it), I get the error: "Token is invalid or expired".

**Reply by grupo-siotec** [reply]
I'm having the same issue, did you manage to transfer the domain @Mic75, @cimply42? @amyegan

**Reply by xcv367** [reply]
I am also experiencing "Token is invalid or expired" after trying to transfer domain to another user. We've tried 5 times and he gets the same issue on all mobile browsers.

**Reply by cimply42** [reply]
@grupo-siotec @xcv367 we managed to resolve this issue by transferring the domain to a team instead of a user. So instead of entering the user ID to transfer the domain to, try entering their Team ID.

**Reply by xcv367** [reply]
@ cimply42 thanks for the feedback, does this require the user to have a Pro subscriptiom?

**Reply by mweiss237** [reply]
Same issue over here. I'm the owner of both accounts which I want to transfer the domain.
I'm pretty sure that is not an issue with an outdated token as I'm sending the transfer request from my account A to account B with both browser windows opened in front of me. So it's just seconds between sending and trying to accept the request.

It seems like there's a bug while transferring domains to a specific account. 
I'd like to avoid creating a team / subscribing to a pro membership because I have the fear that I'm not able to transfer the domain back to my account afterwards and I end up being locked in a pro membership.

@amyegan can you please have a look at that one again? 

Here's the response I get when clicking the "accept" button in the move request. 
```JSON
{
    "error": {
        "code": "invalid_move_token",
        "message": "Token is invalid or expired",
        "token": "Lgxpd6NIJSReLL...",
        "name": "[domain]"
    }
}
```

**Reply by bk-info** [reply]
Go to Settings and find your **Team URL**. Transferring that way works. You don't need a Pro account.
